doc/nrf/releases/release-notes-2.0.0.rst

```diff
@@ -0,0 +1,1090 @@
+      * Optimize calling ZBOSS API in |NCS| platform.
+        If the ZBOSS API is called in the ZBOSS thread context, processing by the workqueue is now skipped.
+
+  * :ref:`lib_zigbee_fota` library:
+
+    * Added:
+
+      * New :kconfig:option:`CONFIG_ZIGBEE_FOTA_SERVER_DISOVERY_INTERVAL_HRS` Kconfig option to configure the interval between queries for the Zigbee FOTA server.
+      * New :kconfig:option:`CONFIG_ZIGBEE_FOTA_IMAGE_QUERY_INTERVAL_MIN` Kconfig option to configure the interval between queries for the available Zigbee FOTA images.
+      * Support for the combined application and network core updates for the nRF5340 SoC.
+
+    * Updated:
+
+      * Download logic to use the :ref:`lib_dfu_multi_image` library API and image structure.
+      * The image generation script by introducing the sub-element structure inside the Zigbee OTA image.
+        Enable :kconfig:option:`CONFIG_ZIGBEE_FOTA_GENERATE_LEGACY_IMAGE_TYPE` to generate images compatible with previous |NCS| releases.
+      * Default value of the :kconfig:option:`CONFIG_ZIGBEE_FOTA_IMAGE_TYPE` to 0x0141.
+      * Setting of the :kconfig:option:`CONFIG_NRF53_ENFORCE_IMAGE_VERSION_EQUALITY` for nRF5340 SoC to ensure integrity of the upgrade image.
+
+    * Removed:
+
+      * The :c:enum:`ZIGBEE_FOTA_EVT_ERASE_PENDING` and :c:enum:`ZIGBEE_FOTA_EVT_ERASE_DONE` events.
+
+  * Fixed:
+
+    * An issue where printing binding table containing group-binding entries results in corrupted output.
+    * An issue where Zigbee shell coordinator would not form a new network after the factory reset operation.
+
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* Added new ``west ncs-sbom`` command that generates :ref:`Software Bill of Materials <west_sbom>`.
+* Added :ref:`bt_fast_pair_provision_script`.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``e86f575f68fdac2cab1898e0a893c8c6d8fd0fa1``, plus some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+* Added :kconfig:option:`CONFIG_NRF53_ENFORCE_IMAGE_VERSION_EQUALITY` to attach the dependencies between application and network core images.
+  This option links the upgrade images in such a way that either both or none is applied.
+* Added support for the Dual-slot execute-in-place (XIP) feature in the |NCS| build system.
+  See the :file:`ncs/nrf/tests/modules/mcuboot/direct_xip` test project for an example of how to leverage this feature in your application.
+
```

Comment:
Upgrades inherited from the upstream MCUboot
boot_serial: Adapt to Zephyr's new CRC APIs (boot_serial implements the serial recovery)
zephyr/boot_serial_extension: us BOOT_LOG instead of LOG_ (boot_serial_extension implements the serial recovery extension)
Loader was reworked so it allows larger minimum flash write size (was 8 B, became 32 B)
boot_serial: zephyr: Add optional timeout to enter serial recovery
zephyr: Use a smaller sha256 implementation
boot_serial: Added support for the echo command
single loader: fixed image decryption for any SoC flash of the pages size which not fitted in 1024 B
removed deprecated DT_CHOSEN_ZEPHYR_FLASH_CONTROLLER_LABEL
Fixed usage of CONFIG_LOG_IMMEDIATE

Reply:
ref.: https://github.com/zephyrproject-rtos/zephyr/pull/46080

Reply:
done